### PR TITLE
ZCS-1417

### PR DESCRIPTION
**Pull request by mihaipopahtp**
- adding Vietnamese translation

### CODE REVIEW

File: tiny_mce/custom/langs/vi.js [hunk 1]
```diff
@@ -0,0 +1,240 @@
+\ufeff/*
```
Comment (tarangkhandelwal) [inline tiny_mce/custom/langs/vi.js]: is this correct? 
+\ufeff at beginning of the file?

Comment (mihaipopahtp) [inline tiny_mce/custom/langs/vi.js]: @tarangkhandelwal 
No, it is not. Sorry for the trouble and thank you for noticing.
That is an UTF-8 BOM encoded in Unicode Escape and it shouldn't be there. Fixing it.

File: WebRoot/messages/ZhMsg_vi.properties [hunk 2]
```diff
@@ -0,0 +1,1958 @@
...
+actionAddressBookDeleted = \u0110\u00e3 x\u00f3a th\u01b0 m\u1ee5c danh b\u1ea1 "{0}".
+actionAddressBookModified = \u0110\u00e3 s\u1eeda th\u01b0 m\u1ee5c danh b\u1ea1 "{0}".
+actionAddressBookRenamed = \u0110\u00e3 \u0111\u1ed5i t\u00ean th\u01b0 m\u1ee5c danh b\u1ea1 "{0}" th\u00e0nh "{1}".
+actionAddressBookTrashed = \u0110\u00e3 chuy\u1ec3n th\u01b0 m\u1ee5c danh b\u1ea1 "{0}" v\u00e0o th\u00f9ng r\u00e1c.
+actionAddressBookMovedToTrash = \u0110\u00e3 chuy\u1ec3n th\u01b0 m\u1ee5c danh b\u1ea1 "{0}" v\u00e0o "Th\u00f9ng r\u00e1c".
+actionBriefcaseCreated = \u0110\u00e3 t\u1ea1o c\u1eb7p t\u00e0i li\u1ec7u "{0}".
+actionBriefcaseCheckConfirm = H\u00e3y ch\u1ecdn h\u1ed9p ch\u1ecdn \u0111\u1ec3 x\u00e1c nh\u1eadn x\u00f3a c\u1eb7p t\u00e0i li\u1ec7u.
+actionBriefcaseColorChanged = M\u00e0u "{0}" c\u1ee7a c\u1eb7p t\u00e0i li\u1ec7u \u0111\u00e3 \u0111\u1ed5i sang "{1}".
+actionBriefcaseDeleted = \u0110\u00e3 x\u00f3a c\u1eb7p t\u00e0i li\u1ec7u "{0}".
+actionBriefcaseModified = \u0110\u00e3 s\u1eeda c\u1eb7p t\u00e0i li\u1ec7u "{0}"".
```
Comment (silentsakky) [inline WebRoot/messages/ZhMsg_vi.properties]: I guess there is one extra double quote at the end of string

File: WebRoot/messages/ZhMsg_vi.properties [hunk 3]
```diff
@@ -0,0 +1,1958 @@
...
+numbers = S\u1ed1
+numberOfItemsPerPage = S\u1ed1 m\u1ee5c hi\u1ec3n th\u1ecb tr\u00ean m\u1ed7i trang
+numberOfContactsToDisplayPerPage = S\u1ed1 l\u01b0\u1ee3ng li\u00ean h\u1ec7 hi\u1ec3n th\u1ecb tr\u00ean m\u1ed7i trang
+oldPassword = M\u1eadt kh\u1ea9u c\u0169
+on = v\u00e0o
+options = T\u00f9y ch\u1ecdn
+optionsAccess = Truy c\u1eadp t\u1eeb \u1ee9ng d\u1ee5ng th\u01b0 kh\u00e1c
+optionsAccessPop = Truy c\u1eadp POP:
+optionsAccessPopDownloadAll = Cho ph\u00e9p t\u1ea3i v\u1ec1 t\u1ea5t c\u1ea3 c\u00e1c th\u01b0, bao g\u1ed3m c\u1ea3 th\u01b0 c\u0169.
+o
```
Comment (silentsakky) [inline WebRoot/messages/ZhMsg_vi.properties]: same as above

File: WebRoot/messages/ZmMsg_vi.properties [hunk 4]
```diff
@@ -0,0 +1,4514 @@
...
+confirmSummary = Th\u01b0 \u0111\u00e3 g\u1eedi: {0}
+contact = Li\u00ean h\u1ec7
+contactCreated = \u0110\u00e3 t\u1ea1o li\u00ean h\u1ec7
+contactGroupToolTip = T\u1ea1o m\u1ed9t nh\u00f3m li\u00ean h\u1ec7 m\u1edbi
+contactList = Danh s\u00e1ch
+contactLists = Danh s\u00e1ch li\u00ean h\u1ec7
+contactPickerHint = Nh\u1eadp t\u00ean c\u1ee7a ng\u01b0\u1eddi nh\u1eadn \u1edf \u0111\u00e2y.
+contactPickerEmailHint = Nh\u1eadp \u0111\u1ecba ch\u1ec9 email c\u1ee7a ng\u01b0\u1eddi nh\u1eadn \u1edf \u0111\u00e2y.
+contactPickerDepartmentHint = Nh\u1eadp ph\u00f2ng c\u1ee7a ng\u01b0\u1eddi nh\u1eadn \u1edf \u0111\u00e2y.
+contactPickerPhoneticHint = Nh\u1eadp t\u00e
```
Comment (silentsakky) [inline WebRoot/messages/ZmMsg_vi.properties]: are we missing starting double quotes around placeholder {0}?

Comment (silentsakky) [inline WebRoot/messages/ZmMsg_vi.properties]: are we missing ending single quote in 'Chuy\u1ec3n ti\u1ebfp ?

Comment (silentsakky) [inline WebRoot/messages/ZmMsg_vi.properties]: are we missing starting single quote?

Comment (silentsakky) [inline WebRoot/messages/ZmMsg_vi.properties]: extra double quote at end of string

File: WebRoot/messages/ZtMsg_vi.properties [hunk 5]
```diff
@@ -0,0 +1,789 @@
...
+errorServerUnavailableLogout = Kh\u00f4ng th\u1ec3 k\u1ebft n\u1ed1i t\u1edbi m\u00e1y ch\u1ee7. \u0110\u00f3ng c\u1eeda s\u1ed5 n\u00e0y \u0111\u1ec3 tho\u00e1t.
+
+account.MAINTENANCE_MODE = T\u00e0i kho\u1ea3n n\u00e0y hi\u1ec7n \u0111ang trong ch\u1ebf \u0111\u1ed9 b\u1ea3o tr\u00ec.
+account.NO_SUCH_ACCOUNT = Kh\u00f4ng c\u00f3 t\u00e0i kho\u1ea3n n\u00e0o nh\u01b0 v\u1eady t\u1ed3n t\u1ea1i.
+account.NO_SUCH_SAVED_SEARCH = Kh\u00f4ng c\u00f3 c\u00e1c t\u00ecm ki\u1ebfm \u0111\u00e3 l\u01b0u n\u00e0o nh\u01b0 v\u1eady t\u1ed3n t\u1ea1i.
+account.ACCT_NO_SUCH_TAG = Kh\u00f4ng c\u00f3 th\u1ebb n\u00e0o nh\u01b0 v\u1eady t\u1ed3n t\u1ea1i.
+
+mail.ALREADY_EXISTS = M\u1ed9t \u0111\u1ed1i t\u01b0\u1ee3ng v\u1edbi t\u00ean \u0111\u00f3 \u0111\u00e3 t\u1ed3n t\u1ea1i.
+mail.MODIFY_CONFLICT = S\u1ed1 phi\u00ean b\u1ea3n xung \u0111\u1ed9t. M\u1ed9t phi\u00ean b\u1ea3n kh\u00e1c \u0111\u00e3 \u0111\u01b0\u1ee3c th\u00eam v\u00e0o gi\u1eefa s\u1eeda v\u00e0 l\u01b0u l\u1ea1i.
+mail.INVALID_NAME = {0}" kh\u00f4ng ph\u1ea3i l\u00e0 t\u00ean h\u1ee3p l\u1ec7. N\u00f3 ch\u1ee9a \u00edt nh\u1ea5t m\u1ed9t k\u00fd t\u1ef1 kh\u00f4ng h\u1ee3p l\u1ec7.
```
Comment (silentsakky) [inline WebRoot/messages/ZtMsg_vi.properties]: same as above

File: WebRoot/messages/ZhMsg_vi.properties (hunk 2 above)
Comment (mihaipopahtp) [inline WebRoot/messages/ZhMsg_vi.properties]: fixed. Thank you for noticing!

File: WebRoot/messages/ZhMsg_vi.properties (hunk 3 above)
Comment (mihaipopahtp) [inline WebRoot/messages/ZhMsg_vi.properties]: fixed. Thank you for noticing!

File: WebRoot/messages/ZmMsg_vi.properties (hunk 4 above)
Comment (mihaipopahtp) [inline WebRoot/messages/ZmMsg_vi.properties]: fixed. Thank you for noticing!

Comment (mihaipopahtp) [inline WebRoot/messages/ZmMsg_vi.properties]: fixed. Thank you for noticing!

Comment (mihaipopahtp) [inline WebRoot/messages/ZmMsg_vi.properties]: fixed. Thank you for noticing!

File: WebRoot/messages/ZtMsg_vi.properties [hunk 6]
```diff
@@ -0,0 +1,789 @@
...
+newTag = Th\u1ebb m\u1edbi
+editFolder = S\u1eeda th\u01b0 m\u1ee5c
+editTag = S\u1eeda th\u1ebb
+folderName = T\u00ean th\u01b0 m\u1ee5c
+tagName = T\u00ean th\u1ebb
+colorLabel = M\u00e0u
+folderLocationLabel = \u0110\u1ecba \u0111i\u1ec3m
+nameMissing = Ph\u1ea3i c\u00f3 t\u00ean.
+nameTooLong = T\u00ean ph\u1ea3i d\u00e0i t\u1ed1i \u0111a {0} k\u00fd t\u1ef1.
+nameInvalid = {0}" kh\u00f4ng ph\u1ea3i l\u00e0 t\u00ean h\u1ee3p l\u1ec7. N\u00f3 ch\u1ee9a \u00edt nh\u1ea5t m\u1ed9t k\u00fd t\u1ef1 kh\u00f4ng h\u1ee3p l\u1ec7.
```
Comment (mihaipopahtp) [inline WebRoot/messages/ZtMsg_vi.properties]: fixed. Thank you for noticing!

File: WebRoot/messages/ZtMsg_vi.properties (hunk 5 above)
Comment (mihaipopahtp) [inline WebRoot/messages/ZtMsg_vi.properties]: fixed. Thank you for noticing!

File: WebRoot/messages/ZhMsg_vi.properties (hunk 2 above)
Comment (silentsakky) [inline WebRoot/messages/ZhMsg_vi.properties]: I think it's till there, maybe your changes are not pushed yet

File: WebRoot/messages/ZtMsg_vi.properties (hunk 6 above)
Comment (silentsakky) [inline WebRoot/messages/ZtMsg_vi.properties]: changes are missing i think

File: WebRoot/messages/ZtMsg_vi.properties (hunk 5 above)
Comment (silentsakky) [inline WebRoot/messages/ZtMsg_vi.properties]: changes are missing i think

**Pull request merged**